ios/app/brave_core_main.mm

```diff
@@ -516,6 +520,35 @@ + (bool)initializeICUForTesting {
   return base::i18n::InitializeICU();
 }
 
++ (void)initializeResourceBundleForTesting {
```

Comment:
<img src="https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExdmg0bXVsa3F3YjQ1aXJ4aWZ5NTg1d3llemw5ODN4aXhsODIzeDdzNSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3o6fJd4EpKyVtfzVUk/giphy.gif" />

Will need to call this in `ScriptExecutionTests` too (maybe some others spots?).